cadence/scripts/kittyItems/get_kitty_item.cdc

```diff
@@ -1,24 +1,76 @@
 import NonFungibleToken from "../../contracts/NonFungibleToken.cdc"
+import MetadataViews from "../../contracts/MetadataViews.cdc"
 import KittyItems from "../../contracts/KittyItems.cdc"
 
-pub struct AccountItem {
+pub struct KittyItem {
+  pub let name: String
+  pub let description: String
+  pub let thumbnail: String
+
   pub let itemID: UInt64
-  pub let typeID: UInt64
   pub let resourceID: UInt64
+  pub let kind: KittyItems.Kind
+  pub let rarity: KittyItems.Rarity
   pub let owner: Address
 
-  init(itemID: UInt64, typeID: UInt64, resourceID: UInt64, owner: Address) {
+  init(
+    name: String,
+    description: String,
+    thumbnail: String,
+    itemID: UInt64,
+    resourceID: UInt64,
+    kind: KittyItems.Kind,
+    rarity: KittyItems.Rarity,
+    owner: Address,
+  ) {
+    self.name = name
+    self.description = description
+    self.thumbnail = thumbnail
+
     self.itemID = itemID
-    self.typeID = typeID
     self.resourceID = resourceID
+    self.kind = kind
+    self.rarity = rarity
     self.owner = owner
   }
 }
 
-pub fun main(address: Address, itemID: UInt64): AccountItem? {
+pub fun dwebURL(_ file: MetadataViews.IPFSFile): String {
```

Comment:
Would it make sense to include methods like this in the `MetadataViews` contract or maybe in some sort of other utility contract?